docs/development/local_setup.md

```diff
@@ -69,31 +112,84 @@ $ git clone git@github.com:gardener/gardener.git
 $ cd gardener
 ```
 
-# Local development
+### Start the Gardener
 
 :warning: Before you start developing, please ensure to comply with the following requirements:
 
 1. You have understood the [principles of Kubernetes](https://kubernetes.io/docs/concepts/), and its [components](https://kubernetes.io/docs/concepts/overview/components/), what their purpose is and how they interact with each other.
 1. You have understood the [architecture of the Garden](https://github.com/gardener/documentation/wiki/Architecture), and what the various clusters are used for.
 
-The development of the Gardener could happen by targetting any cluster. You basically need a Garden cluster (e.g., a [Minikube](https://github.com/kubernetes/minikube) cluster) and one Seed cluster per cloud provider and per data center/region. You can configure the Gardener controller manager to watch **all namespaces** for Shoot manifests or to only watch **one single** namespace.
+The development of the Gardener could happen by targeting any cluster. You basically need a Garden cluster (e.g., a [Minikube](https://github.com/kubernetes/minikube) cluster) and one Seed cluster per cloud provider and per data center/region. You can configure the Gardener controller manager to watch **all namespaces** for Shoot manifests or to only watch **one single** namespace.
 
-In order to ensure that a specific Seed cluster will be chosen, add the `.spec.cloud.seed` field (see [here](../../example/shoot-azure.yaml#L10) for an example Shoot manifest).
+The commands bellow will configure your `minikube` with the absolute minimum resources to launch Gardener API server & controller manager:
+
+```bash
+$ minikube start
+Starting local Kubernetes v1.8.0 cluster...
+[...]
+kubectl is now configured to use the cluster.
+```
 
-## Get started
+The Gardener exposes the API servers of Shoot clusters via Kubernetes services of type `LoadBalancer`. In order to establish stable endpoints (robust against changes of the load balancer address), it creates DNS records pointing to these load balancer addresses. They are used internally and by all cluster components to communicate.
+You need to have control over a domain (or subdomain) for which these records will be created.
+Please provide an *internal domain secret* (see [this](../../example/secret-internal-domain.yaml) for an example) which contains credentials with the proper privileges. Further information can be found [here](../deployment/configuration.md).
 
-Please take a look at the [example manifests folder](../../example) to see which resource objects you need to install into your Garden cluster.
+```bash
+$ mkdir -p dev
+$ cp example/secret-internal-domain.yaml dev/secret-internal-domain.yaml
+# <Put your credentials in dev/secret-internal-domain.yaml>
+$ kubectl apply -f dev/secret-internal-domain.yaml
```

Comment:
Hm, we missed that we need to create the `garden` namespace before applying the internal domain secret manually.
Please add
```bash
$ kubectl apply -f example/namespace-garden.yaml
namespace "garden" created
```
before the other `$ kubectl apply ...` command.